docs/user-guide/configure-overview.md

```diff
@@ -12,192 +12,161 @@ using environment variables with the same name as the property.
-      clientWorkerThreadsCount: 0
-
-      # Worker threads are those performing all the asynchronous I/O operation on the inbound channel.
-      # 0 -> availableProcessors / 2 threads will be used
-
-      workerThreadsCount: 0
-
-      tcpNoDelay: true
-      nioReuseAddress: true
-      nioKeepAlive: true
-      maxInitialLength: 4096
-      maxHeaderSize: 8192
-      maxChunkSize: 8192
-      maxContentLength: 65536
-
-
-      # A timeout for idle persistent connections, in milliseconds.
-      keepAliveTimeoutMillis: 12000
-
-      # Time in milliseconds Styx Proxy Service waits for an incoming request from client
-      # before replying with 408 Request Timeout.
-      requestTimeoutMillis: 12000
-
-
-    admin:
-      connectors:
-      - type: http
-        # Server port for Styx admin interface.
-        port: 9000
-
-      # Number of threads for handling incoming connections on admin interface. 0 -> availableProcessors / 2 threads will be used.
-      bossThreadsCount: 1
-
-      # Number of worker threads for admin interface
-      # Worker threads are those performing all the asynchronous I/O operation on the inbound channel.
-      # 0 -> availableProcessors / 2 threads will be used
-      workerThreadsCount: 1
-
-      tcpNoDelay: true
-      nioReuseAddress: true
-      nioKeepAlive: true
-      maxInitialLength: 4096
-      maxHeaderSize: 8192
-      maxChunkSize: 8192
-      maxContentLength: 65536
-      
-      # Whether to cache the generated JSON for the /admin/metrics and /admin/jvm pages
-      metricsCache:
-        enabled: true
-        expirationMillis: 10000
-
-    loadBalancing:
-      strategy: #Check load balancing documentation for all the possible strategies
-        factory: {class: "com.hotels.styx.client.loadbalancing.strategies.PowerOfTwoStrategy$Factory"}
- 
-
-
-      adaptive:
-        # Adaptive loadbalancer warm-up count. The count is the number of requests that has to hit the
-        # load balancer before it upswitches from round robin strategy to least response time strategy.
-        warmupCount: 100
-
-    metrics:
-      graphite:
-        enabled: true
-        # Host of the Graphite endpoint. Overrides the property from CoreFoundation
+# A string uniquely identifying the host running the application, must be different for all running instances of the application
+# the default value is suitable only for non clustered environments
+jvmRouteName: "${jvm.route:noJvmRouteSet}"
+
+proxy:
+  connectors:
+    http:
+      # Server port for Styx proxy.
+      port: 8080
+    https:
+      port: 8443
+      sslProvider: OPENSSL
+      sessionTimeoutMillis: 300000
+      sessionCacheSize: 20000
+      protocols:
+        - TLSv1.2
+      cipherSuites:
+       - TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384
+       - TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256
+       - TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384
+       - TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256
+  #  0 -> availableProcessors / 2 threads will be used
```

Comment:
Do we want to explain what this value is for?  Also is this 0 ->  comment true or is that the case only when bossThreadsCount is null?

Reply:
This chunk of config is actually not new, but the same as before, only Git diff thinks it is new. Still, we can benefit from adding some explanation.

Reply:
Yes, I noticed it was not new but it seems something we could consider 'bringing the documentation up-to-date'.